### DISCUSSION TITLE
Need help with nextjs monorepo build with docker image

### DISCUSSION THREAD

**Discussion by charlesjiang-greengate** [vercel/next.js#50891] [Help]
### Summary

I have setup a turbo mono repo with nextjs app in the `apps` directory.  The directory structure is as following

```
root
|___ apps
        |___ customer-portal
        |___ client-portal
```

I am able to run `next dev` without issue, but the docker image doesn't work.  Inspecting the `.next` directory found that by turning on output file tracing, the `standalone` directory in `.next` also has the `apps\customer-portal` directory structure as  well, e.g.

```
root
|___ apps
        |___ customer-portal
               |__ .next
                     |__ standalone
                          |__ apps
                                |__ customer-portal
                                      |__ .next
                                      |__ server.js
                                      |__ node_modules
.... etc
```

when docker image starts, it cannot find the `server.js` and therefore failed to start the image. However the `server.js` is in `apps\customer-portal`, maybe as the result of `outputFileTracing` flag on. 

### Additional information

```js
npx next info

    Operating System:
      Platform: darwin
      Arch: arm64
      Version: Darwin Kernel Version 22.4.0: Mon Mar  6 20:59:58 PST 2023; root:xnu-8796.101.5~3/RELEASE_ARM64_T6020
    Binaries:
      Node: 18.15.0
      npm: 9.6.4
      Yarn: N/A
      pnpm: 7.15.0
    Relevant packages:
      next: 13.4.4
      eslint-config-next: 13.4.4
      react: 18.2.0
      react-dom: 18.2.0
      typescript: 5.0.4
```

And my next.config.js as follows

/** @type {import('next').NextConfig} */
const path = require('path');

const nextConfig = {
  reactStrictMode: true,
  compiler: { styledComponents: true },
  webpack(config) {
    config.module.rules.push({
      test: /\.svg$/i,
      issuer: /\.[jt]sx?$/,
      use: ['@svgr/webpack'],
    });
    return config;
  },
  experimental: {
    outputFileTracingRoot: path.join(__dirname, '../../'),
    externalDir: true,
  },
  output: 'standalone',
  outputFileTracing: true,
};

module.exports = nextConfig;
```


### Example

_No response_

**Comment by dz3n** [comment]
What is your Dockerfile?
